datasets/datasets_openml.py

```diff
@@ -46,8 +49,16 @@ def get_data(self):
             self.dataset
```

Comment:
This retrieves the dataset number from its name:
```suggestion
            DATASET[self.dataset]
```